charts/loki-stack/templates/datasources.yaml

```
@@ -9,7 +9,11 @@ metadata:
     chart: {{ template "loki-stack.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+{{- if .Values.grafana.sidecar.datasources.label }}
+    {{ .Values.grafana.sidecar.datasources.label }}: "{{ .Values.grafana.sidecar.datasources.labelValue }}"
+{{- else }}
     grafana_datasource: "1"
+{{- end }}
```

Please indent and quote.

```suggestion
    {{- if .Values.grafana.sidecar.datasources.label }}
    {{ .Values.grafana.sidecar.datasources.label }}: {{ .Values.grafana.sidecar.datasources.labelValue | quote }}
    {{- else }}
    grafana_datasource: "1"
    {{- end }}
```

---

It should be ok now

```bash
Validation success! 👍
==> Linting charts/loki-stack

1 chart(s) linted, 0 chart(s) failed
------------------------------------------------------------------------------------------------------------------------
 ✔︎ loki-stack => (version: "2.8.5", path: "charts/loki-stack")
------------------------------------------------------------------------------------------------------------------------
All charts linted successfully
```

Didn't know about chart-testing, looks amazing!
I will use it again for sure so thanks I guess :)